packages/block-library/src/image/image.js

```diff
@@ -557,6 +571,7 @@ export default function Image( {
 				which causes duplicated image upload. */ }
 			{ ! temporaryURL && controls }
 			{ img }
+			{ isSelected && captionControls }
```

Comment:
The fill is only added when the image block is selected to avoid duplicated controls. When the "Styles" panel is open, or there are other images in the editor.

Probably there's a better way to solve this. 🙇